hibike/unit_tests/hibike_message_tests.py

```diff
@@ -0,0 +1,386 @@
+"""
+Unit tests for functions in hibike_message.
+"""
+
+
+import unittest
+import random
+import queue
+import time
+import threading
+
+import serial
+
+import hibike_message
+import spawn_virtual_devices
+DEVICE_TYPES = list(hibike_message.DEVICES)
```

Comment:
You could probably just use `hibike_message.DEVICES.keys()` where necessary, to avoid instantiating a copy of the keys.